msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -108,9 +71,18 @@
 import org.sakaiproject.util.Validator;
 import org.springframework.orm.hibernate4.HibernateOptimisticLockingFailureException;
 
-import lombok.Getter;
-import lombok.Setter;
-import lombok.extern.slf4j.Slf4j;
+import javax.faces.application.FacesMessage;
+import javax.faces.context.ExternalContext;
+import javax.faces.context.FacesContext;
+import javax.faces.event.ValueChangeEvent;
+import javax.faces.model.SelectItem;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
```

Comment:
I'm not a big fan of using * for imports, I don't want to block the PR for this but let's see what others think

Reply:
Done!